build_deploy.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+IMAGE_BUILD="ekidd/rust-musl-builder:1.30.1"
+
+IMAGE="quay.io/app-sre/cincinnati"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+DOCKERFILE_DEPLOY="dist/Dockerfile"
+
+# clean up
+rm -rf target
+
+docker run --rm -v $(pwd):/home/rust/src $IMAGE_BUILD cargo build --release
+
+docker build -f $DOCKERFILE_DEPLOY -t "${IMAGE}:${IMAGE_TAG}" .
+
+if [[ -n "$QUAY_USER" && -n "$QUAY_TOKEN" ]]; then
```

Comment:
this will be injected by CI so you can assume it is always set on CI

Reply:
TOKEN should be PASSWORD here